docs/resources/samples/convert-csv.md

```diff
@@ -25,38 +25,43 @@ Add the following script and build a flow using the steps given to try the sampl
 
 ```TypeScript
 function main(workbook: ExcelScript.Workbook, csv: string) {
+  let sheet = workbook.getWorksheet("Sheet1");
+
   /* Convert the CSV data into a 2D array. */
```

Comment:
I find the location of this "Convert the CSV data into a 2D array." comment confusing. As a thesis or purpose statement, should it be at the top of the sample, before the new `let sheet = ...` line?